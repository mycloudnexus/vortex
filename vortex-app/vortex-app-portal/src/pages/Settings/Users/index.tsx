import type { ReactElement } from 'react'

const Users = (): ReactElement => {
  return <div>Users</div>
}

export default Users
