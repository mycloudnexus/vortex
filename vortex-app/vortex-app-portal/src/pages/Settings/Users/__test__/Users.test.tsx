import { render } from '@testing-library/react'
import Users from '..'

describe('Users Component', () => {
  it('should render the component', () => {
    const { baseElement } = render(<Users />)
    expect(baseElement).toMatchSnapshot()
  })
})
