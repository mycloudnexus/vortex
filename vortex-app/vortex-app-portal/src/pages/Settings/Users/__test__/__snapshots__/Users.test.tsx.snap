// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Users Component should render the component 1`] = `
<body>
  <div>
    <div>
      Users
    </div>
  </div>
</body>
`;
